lms/static/js/discovery/models/course_discovery.js

```diff
@@ -48,15 +48,16 @@
                     this.meanings[facetKey].terms &&
                     this.meanings[facetKey].terms[termKey] || termKey;
             },
-            parse: function (response) {
+            parse: function (response, termKey) {
```

Comment:
Proponuję zmienić termKey na options (jak to jest w dokumentacji)  https://backbonejs.org/#Model-parse . termKey wykorzystywane jest w innym miejscu w kodzie i wprowadza w błąd, a ta zmienna to po prostu atrybut options modelu backbone

Reply:
@krystianlygas dzięki ! Dostosowałam zmiany :)